aurelia-bootstrap-select/src/util-service.js

```diff
@@ -10,11 +10,15 @@ export class UtilService {
     if (a === null || b === null) return false;
     if (a.length !== b.length) return false;
 
-    // If you don't care about the order of the elements inside
-    // the array, you should sort both arrays here.
-
-    for (let i = 0; i < a.length; ++i) {
-      if (a[i] !== b[i]) return false;
+    //return a.every(_a => b.includes(_a));
+    for (let i = 0; i < a.length; i++) {
+        let aExistsInb = false;
+        for (let j = 0; j < b.length && !aExistsInb; j++) {
+            if (a[i] == b[j])
```

Comment:
I see 3 things to modify for the Linter to pass
1- please add the curly braces to wrap the `if` execution. 
2- can you also change your indentation setting to be 2 spaces, that is the default I use. 
3- double equals `==` will give a Lint warning, I believe it's safe to put triple equals `===` on that line. 
   - If any values are string versus number (like `'1' == 1` is True, while `'1' === 1` is False), I would assume that it's ok to return false on the  `isArrayEqual` question. Is that a good assumption?

Reply:
@ghiscoding I've updated per your requests.  As for the use of identity vs. equality operators, I think Identity should be fine.  The code prior to my update used identity, so I should have as well.